test/asynctest.h

```diff
@@ -21,6 +21,7 @@ void reinit_trace(void);
 
 void tlog(const char *format, ...)
     __attribute__((format(printf, 1, 2)));
+void tlog_empty();
```

Comment:
IMO a function to log a literal string (maybe `tlog_string`) would be better.

Reply:
Changed tlog_empty to tlog_string